SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialDispatcher.cpp

```diff
@@ -69,7 +69,6 @@ void USpatialDispatcher::ProcessOps(Worker_OpList* OpList)
 
 		// Authority Change
 		case WORKER_OP_TYPE_AUTHORITY_CHANGE:
-			StaticComponentView->OnAuthorityChange(Op->authority_change);
```

Comment:
This call has been moved to SpatialReceiver so that it is called at an appropriate time w.r.t the queueing that happens inside critical sections